src/components/flex.js

```diff
@@ -16,6 +16,7 @@ export const Container = styled.div`
 
 export const Heading = styled.div`
   display: flex;
+  position: absolute;
```

Comment:
changed this so the vertical centering would be actually centered

Reply:
This made it such that the "start game" button on the lobby page can't be clicked. I guess it ended up underneath the heading? We can probably just vertically center that page as well as a fix.

Reply:
So I fixed that page, but also noticed that you can't click on the shuffle/video buttons on the main game page either. Probably just need to change the header to be in the corners instead of across the whole top.

Reply:
oops - my b - I will take a look

Reply:
I think I would just explicitly set the zindicies for the gameboard and the header - since we always want the header to be clickable and on top of the board...?

Reply:
Sounds good